transport/server.cc

```diff
@@ -431,9 +432,21 @@ future<foreign_ptr<std::unique_ptr<cql_server::response>>>
             tracing::stop_foreground(trace_state);
         });
         --_server._stats.requests_serving;
+
+        auto exception_handler = overloaded_functor {
```

Comment:
I think (but maybe I'm wrong?) I asked in a previous patch what is this exception_handler, and it is only defined here. So I guess your patches need to be reordered.
Shouldn't this function be in some header file?